README.md

```diff
@@ -30,6 +30,11 @@ The configuration in docker-compose.yml mounts your
 `$GOPATH`. So you can edit code on your host and it will be immediately
 reflected inside Docker images run with docker-compose.
 
+If docker-compose fails with an error message like "Cannot start service
+boulder: oci runtime error: no such file or directory" or "Cannot create
+container for service boulder" you should double check that your `$GOPATH`
+exists and is valid (e.g. does not contain any special characters).
```

Comment:
nm - obviously Docker :-) Looks to be `[a-zA-Z0-9_-]` based on https://github.com/docker/docker/commit/f63cdf0260cf6287d28a589a79d3f947def6a569